interface/forms/ros/templates/ros/general_new.php

```diff
@@ -1,1629 +1,1534 @@
-                                                            <label><input type="radio" name="gastro_pain" value="<?php echo $value;?>"/><?php echo $value;?></label>
-                                                         <?php }
-                                                         }?></td>
-		</tr>
-		<tr>
-			<td class="response_prompt"><?php echo xl("Food Intolerance");?>:</td>
-			<td class="response">                            
-                                                        <?php foreach ($this->form->get_options() as $value) {
-                                                          if($value==$this->form->get_food_intolerance()){?>
-                                                            <label><input type="radio" name="food_intolerance" value="<?php echo $value;?>" checked="checked" /><?php echo $value;?></label>
-                                                         <?php } else {?>
-                                                            <label><input type="radio" name="food_intolerance" value="<?php echo $value;?>"/><?php echo $value;?></label>
-                                                         <?php }
-                                                         }?></td>
-			<td class="response_prompt"><?php echo xl("H/O Hepatitis");?>:</td>
-			<td class="response">                          
-                                                        <?php foreach ($this->form->get_options() as $value) {
-                                                          if($value==$this->form->get_hepatitis()){?>
-                                                            <label><input type="radio" name="hepatitis" value="<?php echo $value;?>" checked="checked" /><?php echo $value;?></label>
-                                                         <?php } else {?>
-                                                            <label><input type="radio" name="hepatitis" value="<?php echo $value;?>"/><?php echo $value;?></label>
-                                                         <?php }
-                                                         }?></td>
-			<td class="response_prompt"><?php echo xl("Jaundice");?>:</td>
-			<td class="response">                            
-                                                        <?php foreach ($this->form->get_options() as $value) {
-                                                          if($value==$this->form->get_jaundice()){?>
-                                                            <label><input type="radio" name="jaundice" value="<?php echo $value;?>" checked="checked" /><?php echo $value;?></label>
-                                                         <?php } else {?>
-                                                            <label><input type="radio" name="jaundice" value="<?php echo $value;?>"/><?php echo $value;?></label>
-                                                         <?php }
-                                           
```

Comment:
The `get_mearche` function is spelled wrong, but upon changing to `get_menarche` the function isn't responsive